pangeo_forge_recipes/storage.py

```diff
@@ -39,8 +42,16 @@ def _copy_btw_filesystems(input_opener, output_opener, BLOCK_SIZE=10_000_000):
                     ) from e
                 if not data:
                     break
-                logger.debug(f"_copy_btw_filesystems copying block of {len(data)} bytes")
                 target.write(data)
+                bytes_read += len(data)
+                elapsed = time.time() - start
+                throughput = int(bytes_read / elapsed)
+                if elapsed // interval >= log_count:
+                    logger.debug(f"_copy_btw_filesystems total bytes copied: {bytes_read}")
+                    logger.debug(
+                        f"avg throughput over {round(elapsed/60, 2)} min: ~{throughput} bytes/sec"
```

Comment:
```suggestion
                        f"avg throughput over {elapsed/60:.2f)} min: {throughput/1_000_000:.2f} MB/sec"
```

Better to use string formatting rather than rounding to get the desired result.